config/boards/nanopc-cm3588-nas.csc

```diff
@@ -0,0 +1,44 @@
+# Rockchip RK3588 octa core 16GB RAM SoC eMMC 4x NVMe 3x USB3 USB2 2.5GbE
+BOARD_NAME="NanoPC CM3588 NAS"
+BOARDFAMILY="rockchip-rk3588"
+#BOARD_MAINTAINER=""
```

Comment:
Sorry, but please uncomment this, otherwise both the build pipeline & auto-maintainer scripts will complain/break. You can leave it empty, the script will fill it in later, but it's gotta be set AFAIK.

Reply:
Did it for ya ;-)